other_recipes/jellof rice.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <h1>Jellof Rice</h1> 
+   <img src="hanxiao-lQN1CVzgQUw-unsplash.jpg"width="50%">
+   <h2>Description</h2>
+   <p>Jellof rice is a well known food eating by every body, it is very delicious when prepared very well</p>
+   <h2>Ingredients</h2>
+   <ul>
+   <li>5 medium sized Roma tomatoes, roughly chopped</li>
+   <li>1 red bell pepper, roughly chopped</li>
+   <li>1 medium sized onion, roughly chopped, set aside</li>
+   <li>2 scotch bonnet peppers (habanero peppers as they are sometimes called)</li>
+  <li>1/4 cup of groundnut oil</li>
+  <li>3 tbsp tomato paste</li>
+  <li>2 cups of parboiled rice</li>
+  <li>2 1/2 cups of chicken stock</li>
+  <li>1 tsp salt to taste</li>
+  <li>1/2 tsp curry powder</li>
+  <li>1/2 tsp thyme</li>
+  <li>1 tsp All purpose seasoning</li>
+  <li>1 Knorr stock cube</li>
+  <li>3 bay leaves</li>
+  <li>water, as needed</li>
+  <ul>
+   <h2>Steps</h2>
+  <Li>Blend your tomatoes and red pepper, scotch bonnet peppers in a food processor or blender for about 45 seconds, make sure that everything is blended well.</Li>
+  <li>In a medium sized pot, heat your oil on medium-high heat. Once the oil is heated add the onions you set aside and fry just until they turn golden brown. Once the onions, have turned brown in color add the tomato paste and fry for 2-3 minutes. Then add the blended tomato mixture (reserve about 1/4 cup and set aside) and fry the mixture with the onions and tomato paste for about 30 minutes. Make sure you stir consistently so that the tomato mixture does not burn.</li>
+  <li>After 30 minutes, turn the heat down to medium, and add the chicken stock. Mix and add your seasonings (salt, curry powder, thyme, all purpose seasoning, and the Knorr stock cube). Continue to boil for 10 minutes.</li>
+  <li>Add the parboiled rice to the pot. Mix it very well with the tomato stew. At this point if you need to add water so that the rice is level with the tomato mixture/chicken stock go ahead and do so. Add the bay leaves, cover the pot, and cook on medium to low heat for 15-30 minutes.</li>
+  <li>When the liquid has almost dried up add the remaining tomato stew, cover, and let it cook for another 5-10 minutes heat until the liquid has completely dried up. Turn off the heat, mix thoroughly, and your Jollof Rice is ready to be eaten!</li>
+
```

Comment:
Where is the parent `ul` or `ol` tags?. Kindly fix.

note that you first `li` element should begin with small letter and not capital letter like you have here.